### DISCUSSION TITLE
login web anime pake oauth github

### DISCUSSION THREAD

**Discussion by arsalgrakhmannn** [deaafrizal/forum#85] [Troubleshooting Room]
saya lg belajar sesi 8 https://www.youtube.com/watch?v=H3BUEO_kuno&list=PLc6SEcJkQ6DwSCyM53kqpM1yNyXT80d3U&index=8
ini kenapa sehabis login malah diarahkan ke http://localhost:3000/users
saya maunya diarahkan ke http://localhost:3000/
terima kasih
![Cuplikan layar 2024-05-20 152402](https://github.com/deaaprizal/forum/assets/56280172/9f6ade19-8fc9-47bd-bafe-56c6624dd79d)

**Comment by FERDIZ-afk** [comment]
Mungkin router.replace nya gan coba di cek
